handler/handler_test.go

```diff
@@ -468,6 +468,56 @@ var expectedCanonicalisedAnnotationsAfterDelete = annotations.Annotations{
 	},
 }
 
+var expectedCanonicalisedAnnotationsAfterAdditon = annotations.Annotations{
+	Annotations: []annotations.Annotation{
+		{
+			Predicate: "http://www.ft.com/ontology/annotation/about",
+			ConceptId: "http://www.ft.com/thing/9577c6d4-b09e-4552-b88f-e52745abe02b",
+		},
+		{
+			Predicate: "http://www.ft.com/ontology/annotation/hasAuthor",
+			ConceptId: "http://www.ft.com/thing/838b3fbe-efbc-3cfe-b5c0-d38c046492a4",
+		},
+		{
+			Predicate: "http://www.ft.com/ontology/annotation/mentions",
+			ConceptId: "http://www.ft.com/thing/0a619d71-9af5-3755-90dd-f789b686c67a",
+		},
+		{
+			Predicate: "http://www.ft.com/ontology/annotation/mentions",
+			ConceptId: "http://www.ft.com/thing/100e3cc0-aecc-4458-8ebd-6b1fbc7345ed",
+		},
+		{
+			Predicate: "http://www.ft.com/ontology/hasDisplayTag",
+			ConceptId: "http://www.ft.com/thing/9577c6d4-b09e-4552-b88f-e52745abe02b",
+		},
+	},
+}
+
+var expectedCanonicalisedAnnotationsSameConceptId = annotations.Annotations{
```

Comment:
I don't see an advantage in introducing a new var here.
Let's remove it and use "expectedCanonicalisedAnnotationsBody" wherever it is used.

Reply:
I can't do without it - the test adds a new annotation to the list.

Reply:
Yes, I missed that.